docs/source/size.mdx

```diff
@@ -0,0 +1,140 @@
+# Get the number of rows and the size in bytes
+
+This guide shows you how to use Datasets Server's `/size` endpoint to retrieve a dataset's size programmatically. Feel free to also try it out with [ReDoc](https://redocly.github.io/redoc/?url=https://datasets-server.huggingface.co/openapi.json#operation/getSize).
+
+The `/size` endpoint accepts the dataset name as its query parameter:
+
+<inferencesnippet>
+<python>
+```python
+import requests
+headers = {"Authorization": f"Bearer {API_TOKEN}"}
+API_URL = "https://datasets-server.huggingface.co/size?dataset=duorc"
+def query():
+    response = requests.get(API_URL, headers=headers)
+    return response.json()
+data = query()
+```
+</python>
+<js>
+```js
+import fetch from "node-fetch";
+async function query(data) {
+    const response = await fetch(
+        "https://datasets-server.huggingface.co/size?dataset=duorc",
+        {
+            headers: { Authorization: `Bearer ${API_TOKEN}` },
+            method: "GET"
+        }
+    );
+    const result = await response.json();
+    return result;
+}
+query().then((response) => {
+    console.log(JSON.stringify(response));
+});
+```
+</js>
+<curl>
+```curl
+curl https://datasets-server.huggingface.co/size?dataset=duorc \
+        -X GET \
+        -H "Authorization: Bearer ${API_TOKEN}"
+```
+</curl>
+</inferencesnippet>
+
+The endpoint response is a JSON containing the size of the dataset, as well as each of its configurations and splits. It provides the number of rows, the number of colums (where applicable) and the size in bytes for the different forms of the data: original files, size in memory (RAM) and auto-converted parquet files. For example, the [duorc](https://huggingface.co/datasets/duorc) dataset has 187.213 rows along all its configurations and splits, for a total of 97MB.
```

Comment:
```suggestion
The endpoint response is a JSON containing the size of the dataset, as well as each of its configurations and splits. It provides the number of rows, the number of columns (where applicable), and the size in bytes for the different forms of data:
- the original files, size in memory (RAM)
- the auto-converted Parquet files 

For example, the [duorc](https://huggingface.co/datasets/duorc) dataset has 187,213 rows across its configurations and splits for a total of 97MB.
```

Reply:
```suggestion
The endpoint response is a JSON containing the size of the dataset, as well as each of its configurations and splits. It provides the number of rows, the number of columns (where applicable), and the size in bytes for the different forms of data:
- the original files,
- the size in memory (RAM),
- the auto-converted Parquet files 

For example, the [duorc](https://huggingface.co/datasets/duorc) dataset has 187,213 rows across its configurations and splits for a total of 97MB.
```

Reply:
^ putting `the size in memory (RAM),` in its own bullet point